py/runtime.h

```diff
@@ -45,6 +45,7 @@ typedef enum {
     MP_ARG_BOOL      = 0x001,
     MP_ARG_INT       = 0x002,
     MP_ARG_OBJ       = 0x003,
+    MP_ARG_UINT      = 0x004,
```

Comment:
Let's not modify `py/` at all. Anything we change is something we need to merge when we update. It looks like there is already [`mp_obj_int_get_uint_checked`](https://github.com/adafruit/circuitpython/blob/main/py/objint_mpz.c#L444-L458). (Added in https://github.com/micropython/micropython/commit/176ab99180a95eeac8794828f2f751a696571bb5) You can set the arg type to obj and then check it yourself in shared-bindings.

Reply:
Yes, there is already `mp_obj_int_get_uint_checked` and I'm using it, albeit indirectly. I'll kill `MP_ARG_UINT` and go with `MP_ARG_OBJ` as you suggest.